Signal/src/ViewControllers/OWSSoundSettingsViewController.m

```diff
@@ -0,0 +1,165 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSSoundSettingsViewController.h"
+#import <AVFoundation/AVFoundation.h>
+#import <SignalMessaging/OWSSounds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSSoundSettingsViewController ()
+
+@property (nonatomic) BOOL isDirty;
+
+@property (nonatomic) OWSSound currentSound;
+
+@property (nonatomic) AVAudioPlayer *audioPlayer;
```

Comment:
We need to retain a reference to the `AVAudioPlayer` so that we can ensure only one "demo" is playing at once, and that we stop demoing sounds if we leave this view.